lib/ioutil/log_entry.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ioutil
+
+import (
+	"time"
+)
+
+const logEntryVersion = 1
+
+// LogEntry attaches a timestamp to a log entry
+// easyjson:json
+type LogEntry struct {
+	Log     string    `json:"log"`
+	Time    time.Time `json:"time"`
+	Version int       `json:"version"`
```

Comment:
I think we definitely need different names for those fields, also, they should be short since they are frequently used.

log -> msg
time -> ts
version -> ver - also for first version we can omit this field, we only may need a version marker in the code if there was a major format change that can not be easily detected.

@emlin That is a good concern. I wonder if we rotate log files, we may have a mix of different versions in one file, which probably will require to have version for each line.